src/scripts.js

```diff
@@ -1,23 +1,36 @@
-let randomNumber = (Math.floor(Math.random() * 50));
+let randomNumber = ((Math.ceil(Math.random() * 50)));
 const userRepo = new UserRepository(userData)
-const user = new User(userData[`${randomNumber}`])
-
+const user = new User(userData[`${randomNumber -1}`])
+const hydration = new Hydration(hydrationData, randomNumber)
 let welcomeName = document.querySelector('.user-first-name-js');
 let userFullName = document.querySelector('.user-full-name-js');
-let userId = document.querySelector('.user-id-js');
 let userAddress = document.querySelector('.user-address-js');
 let userEmail = document.querySelector('.user-email-js');
 let userStrideLength = document.querySelector('.user-stride-length-js');
 let userDailyStepGoal = document.querySelector('.user-daily-step-goal-js');
 let userFriends = document.querySelector('.user-friends-js');
 let stepGoalComparison = document.querySelector('.step-goal-comparison-js');
+let waterConsumedByDay = document.querySelector('.water-consumed-by-day-js');
```

Comment:
adding more querySelectors for new HTML elements we are placing innerHTML inside of.